plugins/builds/artifacts/get.js

```diff
@@ -0,0 +1,51 @@
+'use strict';
+
+const jwt = require('jsonwebtoken');
+const joi = require('joi');
+const schema = require('screwdriver-data-schema');
+const uuid = require('uuid/v4');
+
+const idSchema = joi.reach(schema.models.build.base, 'id');
+const artifactSchema = joi.string().label('Artifact Name');
+
+module.exports = config => ({
+    method: 'GET',
+    path: '/builds/{id}/artifacts/{name*}',
+    config: {
+        description: 'Get a single build artifact',
+        notes: 'Redirects to store with proper token',
+        tags: ['api', 'builds', 'artifacts'],
+        auth: {
+            strategies: ['session'],
+            scope: ['user']
+        },
+        plugins: {
+            'hapi-swagger': {
+                security: [{ session: [] }]
+            }
+        },
+        handler: (request, reply) => {
+            const artifact = request.params.name;
+            const buildId = request.params.id;
+
+            const token = jwt.sign({
+                buildId, artifact, scope: ['user']
```

Comment:
Why we sign with `buildId` and `artifact`? are we going to anything with this?

Reply:
We are not doing anything for now/